src/app/_components/Specialists/PriceFilter.js

```diff
@@ -51,19 +49,23 @@ function PricesList() {
       </ul>
       <ClearFilterButton
         clear={() => {
```

Comment:
Can't it be simplified e.g.
clear={priceParam.remove}

Reply:
this may lead to context leaking
changed it to be oneliner tho!